tox.ini

```
@@ -18,8 +18,8 @@ deps =
     coverage
     codecov>=1.4.0
     coveralls
-    djangolts,pytest: django>=1.11.0,<1.12.0
-    djangocurr: django>=2.2.0
+    djangolts,pytest: django>=3.2.0,<4.0.0
+    djangocurr: django>=4.0a1,<4.1.0
```

I suggest to use `~` notation and use `3.2.9` that add support for `python3.10`:
```suggestion
    djangolts,pytest: django~=3.2.9
    djangocurr: django>=4.0rc1,<4.1.0
```